modules/ROOT/pages/transaction-management.adoc

```diff
@@ -147,6 +147,12 @@ When running in a transactional scope, the entire process must run in the same t
 
 Any other router or scope (for example, Foreach) continues with the transaction and its behavior is not modified.
 
+=== Execution Engine Processing Type Selection
+
+Event processors provide hints to the runtime on which xref:execution-engine.adoc[Processing Type] should be used to perform their task. As transaction scopes are set to run on the same thread, the Processing Type hint will be ignored.
+
+In other words, when processing a flow outside of a transaction, operations in the flow hints the runtime about the Execution Engine Processing Type Selection. When processing a flow inside a transaction, the operations hints are ignored at all and all the transaction is executed in the same Execution Engine Processing Type.
```

Comment:
```suggestion

Event processors provide information to Mule runtime engine about which xref:execution-engine.adoc[Processing Type] is best to execute the required task, and then Mule executes the processor in the corresponding thread. However, this behavior changes when there's a transaction scope in a flow:

* Event processors outside of a transaction scope
+
When Mule executes processors outside of a transaction scope, the Processing Type selection is honored.
* Event processors inside of a transaction scope
+
Because transaction scopes run on the same thread, Mule ignores information about the Processing Type when executing processors as part of a transaction.
```

Reply:
👍  for me